packages/formula-editor/src/utils/parser.ts

```
@@ -0,0 +1,408 @@
+
+    lexedRPN.forEach((symbol) => {
+      let parsedLeftExpression: string, parsedRightExpression: string;
+
+      // If we encounter a number or a variable in the RPN, it is itself
+      // a calculated entity (say a result in itself), needs no modification
+      // and can be directly put into the result stack.
+
+      if (
+        ((symbol[0] === "+" || symbol[0] === "-") && this.variables.has(symbol.substring(1))) || 
+        this.variables.has(symbol) ||
+        (!isNaN(parseFloat(symbol)) && isFinite(parseFloat(symbol)))
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)